docs/api/core/Raycaster.html

```diff
@@ -143,12 +143,14 @@
         [page:Vector3 point] – point of intersection, in world coordinates<br />
         [page:Face3 face] – intersected face<br />
         [page:Integer faceIndex] – index of the intersected face<br />
-        [page:Array indices] – indices of vertices comprising the intersected face<br />
+        [page:Array indices] or [page:Integer index] – indices of vertices comprising the intersected face, or a single vertex index (when intersecting [page:Points])<br />
```

Comment:
I created another pull request (https://github.com/mrdoob/three.js/pull/9654), which includes the proposed change for `Points.raycast` and `Line.raycast`, adds `distanceToRay` in the output of `Line.raycast` and updates the docs. Please discard this pull request in favor of the new one.